Tests/UnitTests/Networking/Backend/BaseBackendPostReceiptDataTestClass.swift

```diff
@@ -17,10 +17,15 @@ import XCTest
 
 @testable import RevenueCat
 
-class BackendPostReceiptDataTests: BaseBackendTests {
+class BaseBackendPostReceiptDataTestClass: BaseBackendTests {
 
-    override func createClient() -> MockHTTPClient {
-        super.createClient(#file)
+    override func invokeTest() {
+        guard Self.self != BaseBackendPostReceiptDataTestClass.self else {
+            print("Ignoring base class \(Self.self)")
+            return
```

Comment:
I guess these tests are already ignored in the setUp?

Reply:
Hah, nope. These base classes are still treated as tests because they inherit from `XCTestCase`

Reply:
I did update them just now to not use `invokeTest`, it's better.

Reply:
So Xcode still runs the tests? :/